sdk/storage/azure-storage-queue/azure/storage/queue/_shared/response_handlers.py

```diff
@@ -83,16 +83,18 @@ def return_context_and_deserialized(response, deserialized, response_headers):
     return response.http_response.location_mode, deserialized
 
 
+def return_raw_deserialized(response, *_):
+    return response.http_response.location_mode, response.context[ContentDecodePolicy.CONTEXT_NAME]
+
+
 def process_storage_error(storage_error):   # pylint:disable=too-many-statements
     raise_error = HttpResponseError
     serialized = False
-    if not storage_error.response or storage_error.response.status_code in [200, 204]:
```

Comment:
Chatted offline about looking into this -- 
Going to revert this to how it was previously as it's been like that since even before this PR that moved it: 
https://github.com/tasherif-msft/azure-sdk-for-python/commit/397c70b4e4e384bde14d3777fe81eef5e64f4d63